### PR TITLE
[feature4]Update new-feature

### PR DESCRIPTION
**Pull request by sowmya333-gif**
i altered new-file.txt

### CODE REVIEW

File: new-feature [hunk 1]
```diff
@@ -5,3 +5,7 @@ new file is edited.... in feature branch
 
 
 edited in feature-1 branch
```
Comment (sowmya333-gif) [inline new-feature:7]: ok do some

**Review by sowmya333-gif — COMMENTED** [review]
hi change this

File: new-feature (hunk 1 above)
Comment (Kysowmya) [inline new-feature:7]: ok good

Comment (Kysowmya) [reply new-feature:7]: chaged in new-fil.txt

**Review by Kysowmya — APPROVED** [review]
approved, good

**Comment by Kysowmya** [comment]
Resolve

**Pull request merged**